src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.managed.cs

```diff
@@ -10,13 +10,27 @@
 using Windows.UI.Xaml.Controls.Primitives;
 using static System.Math;
 using static Windows.UI.Xaml.Controls.Primitives.GeneratorDirection;
+using Uno.UI.Extensions;
+#if __MACOS__
+using AppKit;
+#elif __IOS__
+using UIKit;
+#endif
+#if __IOS__ || __ANDROID__
+using _Panel = Uno.UI.Controls.ManagedItemsStackPanel;
+#else
+using _Panel = Windows.UI.Xaml.Controls.Panel;
+#endif
 
 namespace Windows.UI.Xaml.Controls
 {
+#if __IOS__ || __ANDROID__
+	public abstract partial class ManagedVirtualizingPanelLayout : DependencyObject
```

Comment:
Should this type be internal ?

Reply:
It can be, I've made it so. 

Note that the `ManagedItemsStackPanel` needs to be public or it's not very useful, I've marked it not-`EditorBrowsable` however.